rllib/env/wrappers/unity3d_env.py

```diff
@@ -313,7 +313,7 @@ def get_policy_configs_for_game(
                             action_spaces["Striker"], {}),
             }
 
-            def policy_mapping_fn(agent_id):
+            def policy_mapping_fn(*, agent_id, episode, **kwargs):
```

Comment:
Also here too

Reply:
Yeah, good point, I think I'll remove it here.
I do like these *'s to enforce named args to be used. Makes things much more explicit. But I agree, here it would be too much.